### PR TITLE
Run a round of Centaur during travis. Closes a long national nightmare

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: travis_run_centaur.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Build and run centaur
+echo "Cloning Centaur"
+git clone https://github.com/broadinstitute/centaur.git
+
+cd centaur
+
+echo "Running Centaur with ${PARALLELISM_FACTOR}-way parallelism"
+./run_tests_parallel.sh "${PARALLELISM_FACTOR}"
+
+exit 0
```
Comment (geoffjentry) [inline travis_run_centaur.sh]: If anyone looks before I fix this, this kinda ruins the point :)

Comment (geoffjentry) [inline travis_run_centaur.sh]: In fact the exit trap is gonna make that not so good altogether

**Comment by geoffjentry** [comment]
Note that the travis failure is a good thing (in that the change is working, not that it's an awesome circumstance), as it was due to not all of the centaur tests passing

**Comment by geoffjentry** [comment]
And I still need to run the script through that linter that @kshakir made me use the last time

**Comment by cjllanwarne** [comment]
The centaur tests actually failed because `Futures timed out after 10000ms`. Since we're trying to check correctness rather than Local backend scalability performance here, can we run this using the script?

**Comment by geoffjentry** [comment]
It is using the script, but with 3-way parallel. Trying it with 1 but can't go any lower than that.

The travis machines are jankier than what we normally run on, but IMO that's a good thing here.

I'd also question (worth chatting about, as I don't even know what I think) if correctness vs parallel is really the right thing to be testing here.

File: centaur.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "centaur.centaur.cromwell_branch":"BRANCH",
+  "centaur.centaur.conf":"gs://cloud-cromwell-dev/travis-centaur/jes.conf",
+  "centaur.centaur.pem":"gs://cloud-cromwell-dev/travis-centaur/cromwell-account.pem"
```
Comment (cjllanwarne) [inline centaur.json]: I take it this is appropriately ACL'd?

I was able to gsutil cat it, FWIW

File: travis.wdl [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        cpu: "32"
+        memory: "10 GB"
+        zones: "us-central1-b"
+        failOnStderr: false
+    }
+}
+
+workflow centaur {
+    call centaur
+}
```
Comment (cjllanwarne) [inline travis.wdl]: Insufficient newline

File: travis.wdl [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    >>>
+
+    output {
+        File cromwell_log = "/cromwell_root/logs/cromwell.log"
+        File centaur_log_file = "/cromwell_root/logs/centaur.log"
+    }
+
+    runtime {
+        docker: "geoffjentry/centaur-cromwell:latest"
+        cpu: "32"
```
Comment (cjllanwarne) [inline travis.wdl]: Really necessary? That's mildly concerning

**Pull request closed without merging**